index.js

```diff
@@ -109,23 +106,26 @@ function defaultTemplate({
 function generateCSSReferences({
 	files = [],
 	publicPath = '',
-	cssAttributes = '',
+	attributes = {},
 }) {
+	attributes = generateAttributes(attributes);
+
 	return files
 		.map(
-			file =>
-				`<link href="${publicPath}${file}" rel="stylesheet"${cssAttributes}>`
+			file => `<link href="${publicPath}${file}" rel="stylesheet"${attributes}>`
 		)
 		.join('');
 }
 
 function generateJSReferences({
 	files = [],
 	publicPath = '',
-	jsAttributes = '',
+	attributes = {},
 }) {
+	attributes = generateAttributes(attributes);
```

Comment:
Let's no shadow the function argument to make code easier to understand.

Reply:
Thanks @sapegin! In the current release *v1.0.0*, `generateCSSReferences` accept two arguments `(css, publicPath)`, in this pull request I changed it to accept an object `({files: css, attributes: cssAttributes, publicPath})`. Same situation for `generateJSReferences`. This is a breaking change for me. Do you think it should be released under *v1.1.0* or *2.0.0*?